Sources/LiveViewNative/Modifiers/Layout Adjustments Modifiers/FrameModifier.swift

```diff
@@ -59,11 +59,11 @@ enum FrameModifier: ViewModifier, Decodable, Equatable {
         case alignment
         case width
         case height
-        case minWidth = "min_width"
-        case idealWidth = "ideal_width"
-        case maxWidth = "max_width"
-        case minHeight = "min_height"
-        case idealHeight = "ideal_height"
-        case maxHeight = "max_height"
+        case minWidth
+        case idealWidth
+        case maxWidth
+        case minHeight
+        case idealHeight
+        case maxHeight
     }
```

Comment:
Removing these custom enum strings is what would need to be done across the project at the same time as the switch to using [JSONDecoder.KeyDecodingStrategy.convertFromSnakeCase](https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase)